extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -17,7 +17,7 @@ import { DacFxDataModel } from './api/models';
 import { BasePage } from './api/basePage';
 
 const localize = nls.loadMessageBundle();
-
+const serviceProvider = 'MSSQL';
```

Comment:
readability nit: Since this variable points to MSSQL, I would encode that in the name: MsSqlProvider